packages/create-cli/src/lib/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const NX_JSON_FILENAME = 'nx.json';
+export const NX_JSON_CONTENT = JSON.stringify({
+  $schema: './node_modules/nx/schemas/nx-schema.json',
+  releaseTagPattern: 'v{version}',
```

Comment:
Why include this? Version tags don't seem relevant.

Reply:
it was the output of the generateWorkspace command. I copied it over.
removed the `releaseTagPattern` now